ge/views.py

```diff
@@ -100,6 +101,21 @@ def edit_librarydata(request: HttpRequest, item_id: int) -> HttpResponse:
     return render(request, "ge/edit_item.html", {"form": form})
 
 
+@login_required(login_url="/login/")
+def add_fund(request: HttpRequest) -> HttpResponse:
+    if request.method == "POST":
+        form = LibraryDataEditForm(request.POST)
+        if form.is_valid():
+            new_fund = form.save()
+            messages.success(request, "Fund added successfully.")
```

Comment:
Also for refactoring consistency, please add a success message to the `edit_librarydata` view.  And the `TODO` comment in that view can be removed now that it's done.